README.md

```diff
@@ -5,7 +5,8 @@
 # Subnautica Nitrox
 An open-source, multiplayer modification for the game <a href="https://unknownworlds.com/subnautica/">Subnautica</a>.
 
-![](https://github.com/SubnauticaNitrox/Nitrox/workflows/Building%20Nitrox%20Release/badge.svg)
+[![Build](https://github.com/SubnauticaNitrox/Nitrox/workflows/Building%20Nitrox%20Pull%20Request/badge.svg)](https://github.com/SubnauticaNitrox/Nitrox/actions?query=workflow%3A%22Building+Nitrox+Pull+Request%22)
```

Comment:
The problem is that this link will show the result of the newest pull request and that is not what we want (I think). The current "image not found"-image is just because there hasn't been a release yet.

Reply:
You may be right but as far as I am aware we should still have the image available from last time it ran should it not? Do we even have a action for the release?

Reply:
Yeah this workflow is only for releases: https://github.com/SubnauticaNitrox/Nitrox/blob/master/.github/workflows/buildingRelease.yml
If the action has never run (and that's the case because different workflow) there will be no badge.